app/policies/portal/permission_form_policy.rb

```diff
@@ -0,0 +1,18 @@
+class Portal::PermissionFormPolicy < ApplicationPolicy
+
+  def index?
+    admin_or_manager_project_admin?
```

Comment:
If I understand this method right, it should be called `admin_manager_or_project_admin` instead.